romiscanner/vscan.py

```diff
@@ -56,8 +56,8 @@ def start(self):
             if check_port(port):
                 break
         self.port = port
-
-        proclist = ["romi_virtualscanner", "--", "--port", str(self.port)]
+        logger.critical('came here')
+        proclist = ["romi_bpy",  "romi_virtualscanner", "--", "--port", str(self.port)]
```

Comment:
impose to be in `romiscanner/bin` to work, otherwise fail to find `romi_virtualscanner`...